test/spec/utils/localized-routes.test.js

```diff
@@ -12,12 +12,12 @@ describe('LocalizedRoutes utils', function() {
   });
 
   it('should replace params in url', function() {
-    const route = replaceParams('/foo/:bar/bar/:foo', {bar: 'yolo', foo: 'swag'});
+    const route = replaceParams('/foo/:bar/bar/:foo', { bar: 'yolo', foo: 'swag' });
```

Comment:
Line is too long.<br>'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).